src/commands.rs

```diff
@@ -179,7 +179,7 @@ macro_rules! implement_commands {
                 }
             )*
 
-            /// Incrementally iterate the keys space.  
+            /// Incrementally iterate the keys space.
```

Comment:
These spacing issues aren't related to my change, but they're preventing the build from passing.